src/gc.h

```diff
@@ -113,27 +113,25 @@ typedef struct _mallocarray_t {
 
 // pool page metadata
 typedef struct {
-    struct {
```

Comment:
`data` alignment was for the page itself, not the metadata and I don't think that's ever an issue apart from making sure the `data[]` member is at the beginning of the struct. And yeah, even that is not necessary anymore now that we compute the offset manually (when done right ;-p )